### ISSUE TITLE
ObjectType in http://openmobilealliance.org/tech/profiles/LWM2M.xsd

### ISSUE THREAD

**Issue by MKremserBosch** [OpenMobileAlliance/OMA_LwM2M_for_Developers#58]
What is the meaning of the required attribute "ObjectType" in the LWM2M.xsd.
Can you please describe it in detail.

Many Thanks
Michael

**Comment by jpradocueva** [comment]
@MKremserBosch  
The XML Shema it is used for all the different types of objects that the tool uses. 
ObjectType defines if the object is an standalone object with object header (object ID etc) or if it is a collection of resources when there are no object header. So in the tool when creating OMNA reusable resources then this element is set to one type and if you create an normal object then it is a different type and the values for these are defined in the XML schema.

In short, this attribute is used internally by the LwM2M Editor tool to differentiate if it is an object or a resource.
See new editor tool: http://dev_devtoolkit.openmobilealliance.org/OEditor/default.aspx 

Sorry for the delay, I decided to assist in closing some of the open issues.
